master/buildbot/scripts/checkconfig.py

```diff
@@ -46,8 +46,8 @@ def checkconfig(config):
     if os.path.isdir(configFile):
         basedir = configFile
         try:
-            configFile = getConfigFileFromTac(basedir)
-        except (SyntaxError, ImportError) as e:
+            configFile = getConfigFileFromTac(basedir, quiet=quiet)
+        except (Exception) as e:
```

Comment:
Brackets around `(Exception)` are not needed anymore, since there is only one exception.